docs/release-notes/2.14.0.md

```diff
@@ -0,0 +1,77 @@
+  <colgroup>
+    <col class="field-name">
+    <col class="field-body">
+  </colgroup>
+  <tbody valign="top">
+    <tr class="field-odd field">
+      <th class="field-name">Date:</th>
+      <td class="field-body">January 26, 2021</td>
+    </tr>
+    <tr class="field-even field">
+      <th class="field-name">Installation:</th>
+      <td class="field-body">
+        <a class="reference external" href="https://www.percona.com/software/pmm/quickstart">Installing Percona Monitoring and Management</a></td>
+    </tr>
+  </tbody>
+</table>
+
+Percona Monitoring and Management (PMM) is a free and open-source platform for managing and monitoring MySQL, MongoDB, and PostgreSQL performance.
+
+## Release Highlights
+
+- Switch to push metrics by default
+
+    In PMM v2.12.0, Percona replaced its metrics collection service (formerly Prometheus) with VictoriaMetrics. Historically, PMM used a "pull" method with Prometheus. VictoriaMetrics can operate in either a "pull" or "push" method. When PMM v2.12.0 was released, Percona kept the method as "pull". Now with PMM v2.14.0, Percona is shifting the default to "push". This [blog post](https://www.percona.com/blog/2020/12/01/foiled-by-the-firewall-a-tale-of-transition-from-prometheus-to-victoriametrics/) describes the two methods and why "push" benefits its users. Also, [here is a post](https://www.percona.com/blog/2020/12/16/percona-monitoring-and-management-migration-from-prometheus-to-victoriametrics-faq/) by Peter Zaitzev of FAQs relating to the move to VictoriaMetrics and the "push" model. Documentation on the "push" method is [here](../details/victoria-metrics.md).
+
+- DBaaS Preview phase 1.0 (Technical  Preview)
+
+    DBaaS enables non-DBAs (software architects, developers, site reliability engineers, etc.) to perform typical DBA tasks to manage an organization’s database environment via user interfaces and automation orchestration.
+
+    **Note**: This capability is "feature-flagged" and turned off by default. Users require a variable to be passed to PMM to expose this functionality.
+
+- Pass configured SSL certificates to connect PMM for MongoDB
+
+    Enables PMM administrators to setup configured SSL certificate "keys" to authenticate the connection to PMM, specifically for setting up MongoDB. This is a critical security requirement especially in large enterprise infrastructure environments.
+
+- External services presentation on node summary dashboard
+
+    Improvements to the user experience for adding and viewing external services (any data that can be monitored by a Prometheus export such as: non-Percona supported databases like Reddis, ElasticSearch, Cassandra, etc. or an organization’s external application) on the Node Summary dashboard of PMM.
+
+
+## New Features
+
+* [PMM-5765](https://jira.percona.com/browse/PMM-5765): Add external serverless service (no host or address) with new 'pmm-admin add external-serverless' option
```

Comment:
@rnovikovP this makes no sense (the no host or address) considering you can specify a URL... does this instead mean a PMM server can monitor a remote node that is not running the PMM Client package?

Reply:
We need to explain what 'serverless' means. It's a new term suddenly appearing and not mentioned in the pmm-admin man page.

Reply:
This need to be marked as  Technical Preview

Reply:
Add it to the summary and JIRABOT will extract it

Reply:
Done

Reply:
@PaulJacobs-percona  looks like in Jira the issue  looks good already. can you  check it, pls